src/main/resources/alma/alma.fix

```diff
@@ -19,13 +19,12 @@ paste("id", "~https://lobid.org/resources/", "001", "~#!", join_char: "")
 add_field("@context","https://lobid.org/resources/context.jsonld")
 
 include ("./fix/titleRelatedFields.fix")
-include ("./fix/contribution.fix")
 include ("./fix/describedBy.fix")
 include ("./fix/fields.fix")
 include ("./fix/subjects.fix")
 include ("./fix/item.fix")
 include ("./fix/mediumAndType.fix")
-
+include ("./fix/contribution.fix")
```

Comment:
I like alphabetical ordering, but as we don't force this and it is already unordered, I am gonna ignoring this.

Reply:
This is necessary due to dependencies for https://github.com/hbz/lobid-resources/issues/1282

Reply:
Ah - you mean the ordering _does_ matter ? Ok, I see.